### PR TITLE
Create trigger-dci-apps.yml

### PR DESCRIPTION
**Pull request by tjgresha**
Adding the GHA Trigger to run DCI
Signed-off-by: tjgresha <timothy.gresham@intel.com>

### CODE REVIEW

File: .github/workflows/trigger-dci-apps.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# This is CI trigger for the Intel Redhat DCI-Apps Job
+# It requires a user name and a PAT to access the private repo that contains the actual action.
+# It will trigger on a pull request - can also be run manually 
+
+name: dci-apps- workflow-trigger
```
Comment (hershpa) [inline .github/workflows/trigger-dci-apps.yml:5]: Extra space on line 5 
```suggestion
name: dci-apps-workflow-trigger
```

**Comment by tjgresha** [comment]
Signed-off-by: tjgresha <timothy.gresham@intel.com>

**Comment by tjgresha** [comment]
@hershpa this is a legit CI  failing since the kubeconfig needs to be updated for DCI-Apps

**Comment by hershpa** [comment]
This PR looks good to me. Passing it on to @vbedida79

**Comment by vbedida79** [comment]
lgtm, no changes.

**Comment by hershpa** [comment]
Hi @uMartinXu, we are going to go ahead and merge this PR for testing. We want to test once its merged whether the GitHub Action workflow is working once integrated in repo. When testing thru a fork and a PR, there was a secret issue and thus testing could not be done. We have proved it out on a different test repo already. For more details on secret issue, see https://stackoverflow.com/questions/58737785/github-actions-empty-env-secrets

**Pull request merged**